op.c

```diff
@@ -9815,6 +9815,48 @@ Perl_newCONDOP(pTHX_ I32 flags, OP *first, OP *trueop, OP *falseop)
     return o;
 }
 
+/*
+=for apidoc newTRYCATCHOP
+
+Constructs and returns a conditional execution statement that implements
+the C<try>/C<catch> semantics.  First the op tree in C<tryblock> is executed,
+inside a context that traps exceptions.  If an exception occurs then the
+optree in C<catchblock> is executed, with the trapped exception set into the
+lexical variable given by C<catchvar> (which must be an op of type
+C<OP_PADSV>).  All the optrees are consumed by this function and become part
+of the returned op tree. 
+
+The C<flags> argument is currently ignored.
+
+=cut
+ */
+
+OP *
+Perl_newTRYCATCHOP(pTHX_ I32 flags, OP *tryblock, OP *catchvar, OP *catchblock)
```

Comment:
flags should be unsigned

Reply:
Hrm.. We don't seem to be very consistent with that:

```
leo@shy:~/src/bleadperl/perl [git]
$ grep -c "I32 flags" embed.fnc 
61

leo@shy:~/src/bleadperl/perl [git]
$ grep -c "U32 flags" embed.fnc 
167
```

In particular, every other "new*OP" function uses an I32:

```
ApdR    |OP*    |newCONDOP      |I32 flags|NN OP* first|NULLOK OP* trueop|NULLOK OP* falseop
ApdR    |OP*    |newFOROP       |I32 flags|NULLOK OP* sv|NN OP* expr|NULLOK OP* block|NULLOK OP* cont
ApdR    |OP*    |newLOGOP       |I32 optype|I32 flags|NN OP *first|NN OP *other
...
```

Perhaps for consistency with the other "new*OP" funcs at least, this should stay I32 too?

Reply:
That's 73% U32, which is a pretty significant majority.

Following existing bad practice for the other OP constructors doesn't sway me much

I do think using an unsigned type is better for flags, but since we don't actually have any flags, let along one that might lead to promotion problems, I don't feel a need to insist on changing this.

Reply:
> Following existing bad practice for the other OP constructors doesn't sway me much

> I do think using an unsigned type is better for flags, but since we don't actually have any flags, let along one that might lead to promotion problems, I don't feel a need to insist on changing this.

Perhaps we should do a cleanup of these in a separate commit, then? Go through all our `I32` flags and see what ought to be `U32` instead.